### ISSUE TITLE
Documentation Incorrect

### ISSUE THREAD

**Issue by HeinzKessler** [dotnet/docs#5072]
The statement under "Remarks" is not correct: If the values for Top, Bottom, Left, or Rights were set separately, then All returns -1, even if all 4 properties have the same value:
var p = new Padding( 5 );
// Here, All returns 5
p.Left = 5;
// Now, All returns -1, although all 4 properties still have a value of 5
p.All = 5;
// Now, All returns 5 again

see also: https://referencesource.microsoft.com/#System.Windows.Forms/winforms/Managed/System/WinForms/Padding.cs,6c9dc153b2c496ae

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0c67d24a-2485-92bb-9e66-87104cf3eb61
* Version Independent ID: 9e56ed04-9020-f76c-a783-26ac11dfbee6
* Content: [Padding.All Property (System.Windows.Forms)](https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.padding.all?view=netframework-4.7.1)
* Content Source: [xml/System.Windows.Forms/Padding.xml](https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Windows.Forms/Padding.xml)
* Service: **unspecified**
* GitHub Login: @dotnet-bot
* Microsoft Alias: **dotnetcontent**

**Comment by pkulikov** [comment]
I wonder if it's a bug in documentation or implementation?
Setter of the `Left` property:
```csharp
if (_all || _left != value) {
    _all = false;
    _left = value;
}
```
can be improved like this, for example:
```csharp
if (_left == value) return;
if (_all ) {
    _all = false;
}
_left = value;
```
Though, it doesn't fix situation when from (1,0,1,1) padding we go to (1,1,1,1).

**Comment by HeinzKessler** [comment]
That's right: The implementation is wrong, but I don't think that they will change it, because thats would break people's existing code.

**Comment by mairaw** [comment]
@Tanya-Solyanik can you comment on this one? There's an indication that this is a product bug, but let me know if you'd like to just document the behavior as-is.

**Comment by Tanya-Solyanik** [comment]
This might be a product bug, but, like HeinzKessler said, it would be  a breaking change to fix this and the workaround is simple. As is 'Padding.All` getter indicates that padding values were set using the `Padding.All` setting. this information might be useful.

**Comment by pkulikov** [comment]
>As is `Padding.All` getter indicates that padding values were set using the `Padding.All` setting.

Or during construction. Both by the one-parameter constructor and by the four-parameter constructor if it gets all four the same values: `Padding.All` property is also defined (as I read it from the code).
